README.md

```diff
@@ -61,3 +61,7 @@ TRAPpy's home directory:
 	$ nosetests
 
 If the installation is correct all tests will succeed.
+
+# API reference
```

Comment:
We already have this, there's a "Documentation" heading.  Can you submit a patch with only the "Installaion" typo fixed?